### DISCUSSION TITLE
Support SVG loader without webpack

### DISCUSSION THREAD

**Discussion by shlee-zigbang** [swc-project/swc#5497] [Q&A] [answered]
### Describe the feature

I need SVG loader like `babel-plugin-inline-react-svg` or `svgr`.
How to setting SVG loader?
Is ther plan for SVG, if there is not way to load SVG, 

### Babel plugin or link to the feature description

https://github.com/airbnb/babel-plugin-inline-react-svg

**Comment by kwonoj** [comment, marked as answer]
I would consider this belongs to the plugin, as same as suggested babel plugin does.

**Reply by shlee-zigbang** [reply]
I use swc like `npx swc  --config-file .swcApprc app.js -o app.production.js -s`
Can swc use `babel-plugin-inline-react-svg` plugin in this case?

**Reply by kwonoj** [reply]
> Can swc use babel-plugin-inline-react-svg plugin in this case

No. Someone need to port / write plugins for the SWC.

**Reply by shlee-zigbang** [reply]
Okay, thank you.
